src/elfpy/markets.py

```diff
@@ -4,8 +4,12 @@
 TODO: rewrite all functions to have typed inputs
 """
 
-import numpy as np
 import elfpy.utils.time as time_utils
+from elfpy.user import AgentWallet
+from elfpy.utils.bcolors import bcolors
```

Comment:
Have you tested that these colors work on different terminal types?
Is this addition helpful enough to justify maintenance and increased overhead with setting up logging & printing?